src/Uno.UWP/ApplicationModel/Appointments/Appointment.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.ApplicationModel.Appointments
+{
+	public partial class Appointment
+	{
+		public string Location;  
+		public TimeSpan Duration; 
+		public string Details;  
+		public string Subject; 
+		public DateTimeOffset StartTime; 
+
+		public bool AllDay;  
+		public string CalendarId; 

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5452284)